liblangutil/SemVerHandler.cpp

```diff
@@ -265,14 +275,30 @@ unsigned SemVerMatchExpressionParser::parseVersionPart()
 		{
 			c = currentChar();
 			if (v * 10 < v || v * 10 + static_cast<unsigned>(c - '0') < v * 10)
-				BOOST_THROW_EXCEPTION(SemVerError());
+			{
+				BOOST_THROW_EXCEPTION(
+					SemVerError(
+						1465_error,
```

Comment:
You cannot re-use the same error id in multiple locations, just create new (random) error IDs.